src/main/resources/database/schema.sql

```diff
@@ -17,6 +17,7 @@ create table if not exists dataset (
    created_date timestamp with time zone default now(),
    country varchar(35) not null,
    language varchar(3) not null,
+   xslt_transformer_edm_external text,
```

Comment:
Same here, it is not a transformer by itself

Reply:
Changed the name accordingly to the one in DatasetEntity